apollo-router/src/plugins/rhai/engine.rs

```diff
@@ -287,6 +289,16 @@ mod router_json {
     pub(crate) fn value_to_string(x: &mut Value) -> String {
         format!("{x:?}")
     }
+
+    #[rhai_fn(pure, return_raw)]
+    pub(crate) fn json_encode(input: &mut Dynamic) -> Result<String, Box<EvalAltResult>> {
+        serde_json::to_string(input).map_err(|e| e.to_string().into())
+    }
+
+    #[rhai_fn(pure, return_raw)]
+    pub(crate) fn json_decode(input: &mut ImmutableString) -> Result<Dynamic, Box<EvalAltResult>> {
+        serde_json::from_str(input).map_err(|e| e.to_string().into())
+    }
```

Comment:
should this deal with multipart separators ? This will not be usable with deferred responses.